extensions/script-libraries/dropbox/dropbox.livecodescript

```diff
@@ -0,0 +1,5729 @@
+﻿script "com.livecode.script-library.dropbox"
+/*
+Copyright (c) 2016 Gerard McCarthy
+The MIT License (MIT)
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+*/
+
+on revLoadLibrary
+   if the target is not me then
+      pass revLoadLibrary
+   end if
+   
+   insert the script of me into back
+   
+   if the environment contains "development" then
+      set the _ideoverride of me to true
+      revSBAddAvailableLibrary the short name of me, "Dropbox"
+   end if
+end revLoadLibrary
+
+on revUnloadLibrary
+   if the target is not me then
+      pass revUnloadLibrary
+   end if
+   
+   remove the script of me from back
+end revUnloadLibrary
+
+/**
+Implements the Dropbox v2 API.
+Doesn't include business pro API.
+
+Version 2.0.0
+*/
+
+/**
+
+Get information about a user's account.
+
+Syntax: dropboxGetAccount <pAccessToken>, <pUserAccountID>, [<pCallback>]
+
+Description:
+If the callback parameter is not empty the request will be asynchronus and
+when complete the callback will be sent to the object that accessed the API.
+The callback will be sent with three parameters:
+
+- The request ID which will be the value of the it variable after calling the command
+- The HTTP response code
+- The data returned which will be the same data as documented for the it variable in a synchronous request.
+
+
+Parameters:
+pAccessToken: An OAuth2 Access token to access the user's account
+
+pUserAccountID: The user's unique Dropbox ID
+
+pCallback: The handler to call when the request is complete. If empty the command
+will block until complete. Requires tsNet with LiveCode Indy License.
+
+The result: An error if the HTTP request failed
+
+It: For asynchronous requests it will contain a request ID. This may be used with
+tsNet's tsNetGetStatus function. If the request is synchronous and successful the it
+variable will be a UFT8 encoded JSON string in the form:
```

Comment:
"UFT8" -> "UTF-8", probably.

There are quite a few of these so I won't comment on them individually. ;-)